src/hb-cairo.c

```diff
@@ -126,10 +85,10 @@ get_cairo_draw_funcs (void)
   if (!funcs)
   {
     funcs = hb_draw_funcs_create ();
-    hb_draw_funcs_set_move_to_func (funcs, (hb_draw_move_to_func_t) move_to, nullptr, nullptr);
-    hb_draw_funcs_set_line_to_func (funcs, (hb_draw_line_to_func_t) line_to, nullptr, nullptr);
-    hb_draw_funcs_set_cubic_to_func (funcs, (hb_draw_cubic_to_func_t) cubic_to, nullptr, nullptr);
-    hb_draw_funcs_set_close_path_func (funcs, (hb_draw_close_path_func_t) close_path, nullptr, nullptr);
+    hb_draw_funcs_set_move_to_func (funcs, (hb_draw_move_to_func_t) move_to, NULL, NULL);
```

Comment:
Is there a particular reason this a C not C++ file like the rest of HarfBuzz sources?

Reply:
Agreed. I'd need to pull hb-machinery.hh at least to make the funcs creations threadsafe for example.